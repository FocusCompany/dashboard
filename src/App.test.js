import "raf/polyfill";
import initStoryshots from "@storybook/addon-storyshots";

beforeEach(() => spyOn(console, "error"));
initStoryshots();
