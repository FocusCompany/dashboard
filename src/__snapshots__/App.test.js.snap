// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Graphs BarGraph 1`] = `
<div
  id="bar-graph"
  style={
    Object {
      "height": "300px",
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Graphs ColumnGraph 1`] = `
<div
  style={
    Object {
      "height": "300px",
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Graphs HalfPieGraph 1`] = `
<div
  style={
    Object {
      "height": "300px",
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots Graphs PieGraph 1`] = `
<div
  style={
    Object {
      "height": "300px",
      "width": "100%",
    }
  }
/>
`;
